steelthread-0/walkthrough.md

```diff
@@ -97,41 +106,296 @@ Restart the booster services
 - Find the HTTP connection
-- Set the base URL as the url exposed from the messaging front end application
+- Set the base URL as the url exposed from the CRUD REST booster application that we created earlier
 - Click validate to ensure Fuse Ignite can reach the endpoint
 - Click next 
-- Give the connection a name (booster)
+- Give the connection a name (crud booster)
 - Click create
 
 # Creating the integration
-
-- Open the Fuse Ignite console
+- In the Fuse console
 - Click on integrations
 - Click create integration
 - Click on the Enmaase connection
 - Click on the ``` Subscribe for messages ``` option
-- Under Destination Name add ``` work-queue/worker-updates ```
+- Under Destination Name add ``` work-queue/requests ```
 - Choose Queue as the Destination Type
 - Click Next
-- Set the Specify Output Data Type as ``` Type specification not required ``` this is the default by clicking done
-- On the ``` Choose a Finish Connection ``` screen select the messaging booster connection you created
+- Set the Specify Output Data Type as ``` JSON Schema ``` 
+- Add the following schema
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+- On the ``` Choose a Finish Connection ``` screen select the crud booster connection you created
 - Choose ```Invoke URL ``` 
-- In the URL Path set the path to ```/api/data```
-- In the method select ```GET``` and click next
-- Again under Specify Input Data Type leave it as ``` Type specification not required ``` and click done
-- Click publish at the top right
-- Choose the integration name (steel thread)
+- In the URL Path set the path to ```/api/fruits```
+- In the method select ```POST``` and click next
+- Again under Specify Input Data Type set it to ``` JSON Schema ``` 
+- Add the following schema
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+
+```
+Next click the small plus between the two connections on the left hand side then click ``` add step```
+
+- Pick ```data mapper``` from the choices 
+- Map the stock property to the stock property by dragging stock from the left model to the right model
+- Map type to name by dragging type from the left model to name on the right model
+- Click done at the top right
+
+Add some logging steps (optional)
+
+To add some more visibility we can add some logging steps to our integration.
+
+- On the left hand side inbetween the connections you will see small blue boxes with plus symbols. For each of these do the following:
+
+- click it
+- click add step
+- choose log
+- select ```message body```
 
+Once done we should now have 5 steps in our integration. Click publish in the top right.
+
+Name the integration ```steel thread```
+
+
+# Setup CHE to enable editing of the booster
```

Comment:
How do we access Che from here? Che is provisioned in a separate namespace i.e. not in eval

Reply:
Adding example url with placeholder for routing suffix